outsystems.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I think this file should be hidden, since we want to relegate its function to test-related only and not promote its editing for other use cases.

Reply:
I understand your argument but I can't see this file as a test-related stuff only. I think we should start having the required extensibility configuration for an OutSystem Plugin along with a specific version of the Cordova Plugin, since IMHO it will improve troubleshooting too.